pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>de.qytera</groupId>
     <artifactId>qtaf</artifactId>
-    <version>0.0.9</version>
+    <version>LOCAL_2023_04_27_001</version>
```

Comment:
Please don't push `pom` files containing development versions.

Reply:
Updated version to 0.1.0